pkg/crc/systemd/systemd_linux_test.go

```diff
@@ -0,0 +1,98 @@
+package systemd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/code-ready/crc/pkg/crc/systemd/states"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func newMockCommander(t *testing.T) (*Commander, *mockSystemdRunner) {
+	runner := mockSystemdRunner{
+		test: t,
+	}
+
+	return &Commander{
+		commandRunner: &runner,
+	}, &runner
+}
+
+func TestSystemd(t *testing.T) {
```

Comment:
Interesting!

If I was using Java, I would have written something like

```java
when(runner.Run("sysctl", "enable", "foo")).thenReturn(...)
when(runner.Run("sysctl", "enable", "bar")).thenFail(...)

assertThat(systemctl.Enable("foo")).isTrue()
assertThat(systemctl.Enable("bar")).isFalse()
```

With Go, I would have probably set in the stone what the runner should answer.

```go
mockSystemdRunner{
  commands: map[string]bool{
    "systemctl enable foo": true,
    "systemctl enable bar": false,
  }
}
```

Don't change anything! Both tests are good for me :)